meta-resin-common/recipes-support/resin-mounts/resin-mounts/etc-machine-id.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Machine-id Mount Point
+Requires=mnt-conf.mount
+After=mnt-conf.mount
+Before=systemd-machine-id-commit.service
+
+[Service]
```

Comment:
Because this service wants to mount a file and if that location does not exist the normal mount unit will create a directory before attempting to mount. Basically I need to pre run a touch before the mount.